style.css

```diff
@@ -0,0 +1,3 @@
+.header{
+  color: brown;
+}
```

Comment:
:red_circle:  There are some lint errors in this file. Please make sure you fix them